static/js/src/advantage/subscribe/react/components/Form/Feature/Feature.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { FormProvider } from "advantage/subscribe/react/utils/FormContext";
+import Feature from "./Feature";
+import {
+  ProductListings,
+  ProductTypes,
+} from "advantage/subscribe/react/utils/utils";
+import { productListFixture } from "advantage/subscribe/react/utils/test/Mocks";
+
+beforeAll(() => {
+  global.window.productList = productListFixture as ProductListings;
+});
+
+test("Feature section renders correctly", () => {
+  render(
+    <FormProvider>
+      <Feature />
+    </FormProvider>
+  );
+
+  screen.getByText("Select Infra only");
+  screen.getByText("Select Apps only");
+  screen.getByText("Select Infra + Apps");
```

Comment:
I'm just checking they exist so by using getByText it would fail if they didn't.

Reply:
I see, didn't realise in MAAS we'd set up [eslint to complain if there's no explicit assertion](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-explicit-assert.md). I feel like writing out the whole thing is a bit more obvious and readable but since we don't have a team-wide rule or anything then whatevz